libethdrivers/Kbuild

```diff
@@ -13,4 +13,4 @@
 libs-$(CONFIG_LIB_ETHIF) += libethdrivers
 libethdrivers-${CONFIG_LIB_LWIP} += liblwip
 libethdrivers-${CONFIG_LIB_PICOTCP} += libpicotcp
-libethdrivers: ${libethdrivers-y} libutils libsel4 $(libc) libplatsupport common
+libethdrivers: ${libethdrivers-y} libutils libsel4 libsel4utils $(libc) libplatsupport common
```

Comment:
Just a minor question of why libsel4utils needed to be added as a dependency? I don't see any usage of things from it in this commit, were there some pre-existing usages that were broken prior to this being added?

Reply:
I don't remember why this was added. I just removed the dependency and the library builds for both the zynq7000 and the i.MX6 without any issues. This can be removed.